### PR TITLE
storybook: add select story & add CSS fixes

### PR DESCRIPTION
**Pull request by dschaller**
### Description

Add new story for the Select component. This also includes various style fixes and cleanup to Select in order to prevent the label from overlaying on top of the select window when the specified width < label text length.

select

### Testing Performed

manual

### CODE REVIEW

File: frontend/packages/core/src/Input/select.tsx [hunk 1]
```diff
@@ -72,18 +70,21 @@ const Select: React.FC<SelectProps> = ({
   };
 
   React.useEffect(() => {
-    onChange(options[selectedIdx].value || options[selectedIdx].label);
+    onChange(options[selectedIdx]?.value || options[selectedIdx].label);
```
Comment (scarlettperry) [inline frontend/packages/core/src/Input/select.tsx:73]: should it be `?.label` all throughout?

**Comment by scarlettperry** [comment]
noticed that the select with label doesn't align with the fields below in the start-experiment workflow
Screen Shot 2020-10-17 at 10 38 22 PM

**Comment by dschaller** [comment]
Awesome catch! Let me take a deeper look. Thanks :)

File: frontend/packages/core/src/Input/select.tsx (hunk 1 above)
Comment (dschaller) [reply frontend/packages/core/src/Input/select.tsx:73]: every SelectOption should enforce that label is required. Removed the optional chaining of this below

**Comment by dschaller** [comment]
@scarlettperry thanks for catching that issue. It looks like that select isn't meant to be a part of the table fields but instead hovering above. I fixed the CSS to properly set the default width :)

**Review by scarlettperry — APPROVED** [review]

**Pull request merged**